CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -373,6 +373,43 @@
             </StackPanel>
                 
         </ControlTemplate>
+        
+        <ControlTemplate x:Key="CertificateCommands">
+            <StackPanel
+                Orientation="Vertical">
+                <StackPanel
+                    Orientation="Horizontal">
+                    <Button
+                        Margin="4"
+                        Width="48"
+                        Command="{Binding RefreshItem}"
+                        CommandParameter="{Binding SelectedCertificate}"
+                        IsEnabled="{Binding IsAccountConnected, NotifyOnSourceUpdated=True}">
+                        <Button.Content>
+                            <StackPanel>
+                                <Image Source="../Images/refresh.png" Height="36"/>
+                                <TextBlock HorizontalAlignment="Center" Text="Refresh" FontSize="12" Foreground="Black" />
+                            </StackPanel>
+                        </Button.Content>
+                    </Button>
+                    <Button
+                        Margin="4"
+                        Width="48"
+                        Command="{Binding Delete}"
```

Comment:
Delete is existing general-purpose command - after discussion with Pablo, we are okay to ignore this comment